webservices/rest.py

```diff
@@ -43,6 +43,24 @@ def sqla_conn_string():
 api = restful.Api(app)
 db.init_app(app)
 
+# api.data.gov
+trusted_proxies = ('192.168.114.100')
```

Comment:
It looks like `trusted_proxies` is expected to be a tuple, but here it's going to be a string. For a 1-tuple, use
```python
trusted_proxies = ('192.168.114.100', )
```